Can not publish listing: This app has too few screenshots

### Describe the bug

I only have screenshots in the default language and German. The other languages do not have screenshots. When executing `publishListingRelease`, I get the following error message:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:publishReleaseListing'.
> com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
  {
    "code" : 403,
    "errors" : [ {
      "domain" : "androidpublisher",
      "message" : "This app has too few screenshots for language cs-CZ.",
      "reason" : "tooFewScreenshots"
    }, {
      "domain" : "androidpublisher",
      "message" : "This app has too few screenshots for language ru-RU.",
      "reason" : "tooFewScreenshots"
    }, {
      "domain" : "androidpublisher",
      "message" : "This app has too few screenshots for language fr-FR.",
      "reason" : "tooFewScreenshots"
    }, {
      "domain" : "androidpublisher",
      "message" : "This app has too few screenshots for language es-ES.",
      "reason" : "tooFewScreenshots"
    } ],
    "message" : "This app has too few screenshots for language cs-CZ."
  }
```

The Google Play website allows to upload translations for store entries even if they do not have screenshots, as long as the default language has screenshots. Just a wild guess: maybe the plugin does not find local screenshots and provides an empty array to the API but it should rather provide nothing instead.

#### Versions

 - Gradle Play Publisher: `2.3.0-SNAPSHOT`
 - Gradle Wrapper (if applicable): `https://services.gradle.org/distributions/gradle-4.4-all.zip`
 - Android Gradle Plugin (if applicable): I don't know

### Additional context

This is my folder structure:
<details>
<summary>Click to expand</summary>

```
├── contact-email.txt
├── contact-website.txt
├── default-language.txt
├── listings
│   ├── cs-CZ
│   │   ├── full-description.txt
│   │   ├── short-description.txt
│   │   └── title.txt
│   ├── de-DE
│   │   ├── full-description.txt
│   │   ├── graphics
│   │   │   └── phone-screenshots
│   │   │       ├── 00.jpg
│   │   │       ├── 01.jpg
│   │   │       ├── 02.jpg
│   │   │       └── 03.jpg
│   │   ├── short-description.txt
│   │   └── title.txt
│   ├── en-US
│   │   ├── full-description.txt
│   │   ├── graphics
│   │   │   ├── feature-graphic
│   │   │   │   └── featureGraphic.png
│   │   │   ├── icon
│   │   │   │   └── icon.png
│   │   │   └── phone-screenshots
│   │   │       ├── 00.jpg
│   │   │       ├── 01.jpg
│   │   │       ├── 02.jpg
│   │   │       └── 03.jpg
│   │   ├── short-description.txt
│   │   └── title.txt
│   ├── es-ES
│   │   ├── full-description.txt
│   │   ├── short-description.txt
│   │   └── title.txt
│   ├── fr-FR
│   │   ├── full-description.txt
│   │   ├── short-description.txt
│   │   └── title.txt
│   └── ru-RU
│       ├── full-description.txt
│       ├── short-description.txt
│       └── title.txt
```

</details>

--- a/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/GenerateResources.kt
+++ b/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/GenerateResources.kt
@@ -19,7 +19,9 @@ import com.github.triplet.gradle.play.internal.nullOrFull
 import com.github.triplet.gradle.play.internal.orNull
 import com.github.triplet.gradle.play.internal.parents
 import com.github.triplet.gradle.play.internal.playPath
+import org.gradle.api.Action
 import org.gradle.api.DefaultTask
+import org.gradle.api.file.CopySpec
 import org.gradle.api.tasks.CacheableTask
 import org.gradle.api.tasks.InputFiles
 import org.gradle.api.tasks.Internal
@@ -72,7 +74,8 @@ open class GenerateResources : DefaultTask() {
         }
         inputs.removed { project.delete(file.findDest()) }
 
-        for (default in changedDefaults.reversed().distinctBy { it.name }) {
+        val writeQueue = mutableListOf<Action<CopySpec>>()
+        for (default in changedDefaults) {
             val listings = default.findDest().climbUpTo(LISTINGS_PATH)!!
             val relativePath = default.invariantSeparatorsPath.split("$defaultLocale/").last()
 
@@ -82,11 +85,12 @@ open class GenerateResources : DefaultTask() {
                     .filterNot(File::exists)
                     .filterNot(::hasGraphicCategory)
                     .forEach {
-                        project.copy {
+                        writeQueue += Action {
                             from(default).into(File(resDir, it.parentFile.toRelativeString(resDir)))
                         }
                     }
         }
+        writeQueue.forEach { project.copy(it) }
     }
 
     private fun File.validate() {

